src/components/legend/draw.js

```diff
@@ -426,7 +448,7 @@ function setupTraceToggle(g, gd) {
         }
     });
     traceToggle.on('mouseup', function() {
-        if(gd._dragged) return;
+        if(gd._dragged || gd._editing) return;
```

Comment:
I'd like to get @etpinard 's take on this, but my feeling is we don't want to introduce another `gd._flag` - if anything we'd like to get rid of more of these as they're rather brittle and hard to maintain.

Also it's not clear to me that the right course of action is to quit the new handler if we're editing something else. I wonder if it would work to instead make a helper to ensure the edit box's blur handler gets called as it should? Honestly I don't understand why that's not already happening, but what if we just made a helper (as an export of `svg_text_utils`) like:
```
function blurEditable(gd) { d3.select(gd).selectAll('.editable').each(function() { this.blur(); }); }
```

If what the user did was clicked on the editable item accidentally, then wanted to move on to the interaction they *really* intended, quitting here will just add an extra click for them, and `editable.blur()` should be a simple removal of the edit box that doesn't affect the plot, so I see no harm in continuing the new interaction in that case.

I guess there could be weird side-effects if you type something in an edit box, then go click somewhere else, such that before the new handler can execute the plot has to redraw. The most extreme case I think would be if you have the legend off the right side, then you edit some legend text and make it longer, and we auto-expand the plot margin to accommodate it. That results in a very aggressive redraw that might actually delete and recreate the whole rest of the plot (not very d3-idiomatic, I know... we're working on it!). Would need a bit of playing around to see if that would cause any problems...